src/node_buffer.cc

```diff
@@ -1223,6 +1223,14 @@ static void EncodeInto(const FunctionCallbackInfo<Value>& args) {
   results[1] = written;
 }
 
+static void IsUtf8(const FunctionCallbackInfo<Value>& args) {
+  CHECK_GE(args.Length(), 1);
```

Comment:
```suggestion
  CHECK_EQ(args.Length(), 1);
```

Are other arguments acceptable? Otherwise, I recommend changing it to CHECK_EQ.

Reply:
No, you are right. I’ll update! Thanks for the review.